testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/script/UndeployedScriptMapperNotAvailableTest.java

```diff
@@ -101,9 +101,10 @@ public void testMapperNotRecognizedWhenDisabled() throws Exception {
 
             app.getProtocolMappers().createMapper(mapper).close();
         }
-
         deployer.undeploy(SCRIPT_DEPLOYMENT_NAME);
-        assertTrue(app.getProtocolMappers().getMappers().isEmpty());
-        assertTrue(app.getProtocolMappers().getMappersPerProtocol(app.toRepresentation().getProtocol()).isEmpty());
+        reconnectAdminClient();
+        ClientResource cl = findClientResourceByClientId(adminClient.realm("test"), "test-app");
```

Comment:
here we needed a new ClientResource, for the old one would definitely be closed after undeploying/restarting, and the reference to `app` would be stale.